common/lib/xmodule/xmodule/modulestore/__init__.py

```diff
@@ -9,6 +9,8 @@
 import datetime
 from uuid import uuid4
 
+from datetime import datetime
```

Comment:
This import is overriding the one on line 9 which imports the module not the class. It's breaking a bunch of tests.